common/importer/chrome_importer_utils_linux.cc

```diff
@@ -21,6 +21,28 @@ base::FilePath GetChromeUserDataFolder() {
   return result;
 }
 
+base::FilePath GetChromeBetaUserDataFolder() {
+  base::FilePath result;
+  if (!base::PathService::Get(base::DIR_HOME, &result))
+    return base::FilePath();
```

Comment:
I don't have much knowledge about the code here... but I wonder if these functions should rather return `absl::optional<base::File>`, so one is forced to never forget the FilePath check for empty.

Reply:
I think that's a great concern - I don't want to change too much in this PR. Can you capture a follow up for this?

Reply:
Yeah, it might be better to follow up after, specially because there might be other cases not covered by this PR in the same files doing the same.